File: 1974-minimum-time-to-type-word-using-special-typewriter/1974-minimum-time-to-type-word-using-special-typewriter.cs
public class Solution {
    public int MinTimeToType(string word) {
        int n = word.Length;
        int res = n;
        
        for (int i = 0; i < n - 1; i++)
        {
            res += Math.Min(Math.Abs(word[i] - word[i + 1]), 26 - Math.Abs(word[i] - word[i + 1]));
        }
        
        return res + Math.Min(Math.Abs('a' - word[0]), 26 - Math.Abs('a' - word[0]));
    }
}

File: 1974-minimum-time-to-type-word-using-special-typewriter/1974-minimum-time-to-type-word-using-special-typewriter.c
int minTimeToType(char* word) {
    int n = strlen(word);
    int res = n;
    for(int i = 0;i < n - 1;i++){
        res += fmin(abs(word[i] - word[i+1]) , 26 - abs(word[i] - word[i+1]));
    }
    return res + fmin(abs((int) 'a' - word[0]), 26 - abs((int) 'a' - word[0]));
}